documentation/.eslintrc.js

```diff
@@ -1,21 +1,23 @@
 module.exports = {
-  plugins: [
-    'prefer-let',
-  ],
+  plugins: ['prefer-let'],
   overrides: [
     {
       files: ['examples/**/*.ts', 'snippets/**/*.ts'],
       rules: {
         'object-shorthand': ['error', 'never'],
-        '@typescript-eslint/no-unused-vars': ['error', {
-          varsIgnorePattern: 'response|datasourceUrl|MySchema', 
-          argsIgnorePattern: 'context|param'
-        },],
-        'max-len': ['error', { 'code': 80 }],
-        'quotes': ['error', 'double'],
+        '@typescript-eslint/no-unused-vars': [
+          'error',
+          {
+            varsIgnorePattern: 'response|datasourceUrl|MySchema',
+            argsIgnorePattern: 'context|param',
+          },
+        ],
+        'max-len': ['error', {code: 80}],
```

Comment:
80 is too small. can we do 

```
    'max-len': [
      'error',
      {
        code: 120,
        tabWidth: 2,
        ignoreUrls: true,
        ignoreStrings: true,
        ignoreTemplateLiterals: true,
        ignoreRegExpLiterals: true,
        ignorePattern: '^import ',
      },
    ],

```

Reply:
just to make sure, this is for the `documentation` linter that you want to make the change for? cc @ekoleda-codaio

Reply:
oops mb. meant for the main one. and the main one has no configure for max-len. new configurations can be added now or later. I don't have a preference.